kitsune/questions/jinja2/questions/includes/question_editing_frame.html

```diff
@@ -126,21 +116,25 @@ <h2>Do some stuff.</h2>
                     {% endif %}
 
                   <li class="{{ li_class }} {% if field.errors %}has-error invalid{% endif %} cf">
-                    {{ field.label_tag()|safe }}
-
-                    {% if field.name == 'content' %}
-                      {{ content_editor(field) }}
-                    {% elif field.name == 'troubleshooting' %}
-                      {{ troubleshooting_instructions(field) }}
+                    {% if field.name == 'notifications' %}
+                      {{ checkbox(field) }}
                     {% else %}
-                      {{ field|safe }}
+                      {{ field.label_tag()|safe }}
+
+                      {% if field.name == 'content' %}
+                        {{ content_editor(field) }}
+                      {% elif field.name == 'troubleshooting' %}
+                        {{ troubleshooting_instructions(field) }}
+                      {% else %}
+                        {{ field|safe }}
+                      {% endif %}
                     {% endif %}
 
                     {% if field.help_text and field.name != 'troubleshooting' %}
                       <p>{{ field.help_text|safe }}</p>
                     {% endif %}
 
-                    {% if field.name == 'title' %}
+                    {% if field.name != 'content' %}
```

Comment:
This looks weird, excluding just content for errors. What's the desired behavior here?

Reply:
content handles its own errors in the `content_editor` macro